xml/System.Web.Configuration/BrowserCapabilitiesFactory.xml

```diff
@@ -67,12 +67,22 @@
         <Parameter Name="browserCaps" Type="System.Web.HttpBrowserCapabilities" />
       </Parameters>
       <Docs>
-        <param name="ignoreApplicationBrowsers">To be added.</param>
-        <param name="headers">To be added.</param>
-        <param name="browserCaps">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
-      </Docs>
+        <param name="ignoreApplicationBrowsers"><see langword="true" /> to ignore definitions for the A500 browser in application-level browser definition files; otherwise, <see langword="false" />.</param>
+        <param name="headers">The collection of headers included in the current request.</param>
+        <param name="browserCaps">The <see cref="T:system.web.httpbrowsercapabilities"/> object to initialize.</param>
+        <summary>This API supports the .NET Framework infrastructure and is not intended to be used directly from your code. Initializes an <see cref="T:system.web.httpbrowsercapabilities"/> object that represents the capabilities of the A500 browser making the current request.</summary>
+        <remarks>
+        <format type="text/markdown"> <![CDATA[
+
+        ## Platforms
+
+        Windows 7, Windows Vista, Windows XP SP2, Windows XP Media Center Edition, Windows XP Professional x64 Edition, Windows XP Starter Edition, Windows Server 2008 R2, Windows Server 2008, Windows Server 2003, Windows Server 2000 SP4, Windows Millennium Edition, Windows 98
+        The .NET Framework and .NET Compact Framework do not support all versions of every platform. For a list of the supported versions, see [.NET Framework System Requirements](/dotnet/framework/get-started/system-requirements).
+
+        ]]></format>
+
+        </remarks>
+      </Docs> 
```

Comment:
This tag generates the "This API supports the .NET Framework infrastructure and is not intended to be used directly from your code. " boilerplate, instead of adding that text in the summary.

```suggestion
        <forInternalUseOnly> 
      </Docs> 
```